Geometry/HGCalCommonData/interface/HGCalDDDConstants.h

```diff
@@ -102,13 +101,16 @@ class HGCalDDDConstants {
 			  float& tl, float& alpha) const;
   bool waferInLayer(int wafer, int lay) const;
```

Comment:
@bsunanda - please, check that calculations are necessary in the member function. For example, line 806 seems to be equal to hexside_  = 2.0 * rmax_ * tan30deg_;